great_expectations/expectations/metrics/column_aggregate_metrics/column_partition.py

```diff
@@ -154,19 +166,35 @@ def _get_column_partition_using_metrics(bins: int, n_bins: int, _metrics: dict)
         sturges = np.log2(1.0 * nonnull_count + 1.0)
         min_, _25, _75, max_ = _metrics["column.quantile_values"]
 
-        min_original = min_
-        max_original = max_
-
-        datetime_detected: bool = is_ndarray_datetime_dtype(
-            data=[min_, _25, _75, max_], parse_strings_as_datetimes=True
+        original_ndarray_is_datetime_type: bool
+        conversion_ndarray_to_datetime_type_performed: bool
+        data: np.ndaarray
```

Comment:
Not crazy about calling this `data`. Especially since it seems to be different than the `data` being passed into the `convert_ndarray_to_datetime_dtype_best_effort`.

Reply:
Nevermind, I get it now.

Reply:
@NathanFarmer There are full tests for this metric from previous PR.  Thanks.